src/CaBundle.php

```diff
@@ -130,21 +128,17 @@ public static function getSystemCaRootBundlePath(LoggerInterface $logger = null)
      */
     public static function getBundledCaBundlePath()
     {
-        $caBundleFile = __DIR__.'/../res/cacert.pem';
+        $caBundleFile = __DIR__. str_replace('/', DIRECTORY_SEPARATOR, '/../res/cacert.pem');
```

Comment:
This change seems useless

Reply:
Does this work on Windows ? I assumed it would need the backslash ? if not then its fine - I dont have windows to test with

Reply:
Windows has no problem with forward-slashes really so please undo

Reply:
Reverted, is there any use case to use the DIRECTORY_SEPARATOR then ? I always assumed it was exactly this case

Reply:
It is made for the case of cross-platform dev, but it's not strictly needed, at least for windows. There might be oddball systems out there with something else than slash I am not sure :)